### DISCUSSION TITLE
Make innerbrowser actions/functions disableable

### DISCUSSION THREAD

**Discussion by douglasjam** [Codeception/Codeception#6203] [Ideas]
#### What are you trying to achieve?
I have my own custom REST helper, this helper have some functions like seeStatusCode, but since we do already have the InnerBrowser functions by default, they get mixed.

I'd like to have a way to disable the InnerBrowser functions when building my actions. Like that all devs in my project when they try  $I->, I do not get all these functions from InnerBrowser, but just the ones I activated, in this case, my custom REST Helper.

---

Why do I have my custom REST helper, and did not extend the original one? Because I want to debug without going through NGINX, with my custom REST Helper, I instanciate the Symfony Kernel Browser, so the breakpoint can go directly
```php
    /** @var Symfony $symfony */
        $symfony = $this->getModule(ModuleConstant::SYMFONY);

        $this->client = new HttpKernelBrowser($symfony->kernel);
```

---

* Codeception version: 4.1
* PHP Version: 7.4.7
* Operating System: Alpine

**Comment by Naktibalda** [comment]
It is unlikely that such functionality will be implemented any time soon.
